tests/stdlib/tparsecfg.nim

```diff
@@ -1,23 +1,76 @@
 discard """
-  output: '''OK'''
+  targets: "c js"
 """
 
-#bug #6046
-import parsecfg
-
-var config = newConfig()
-config.setSectionKey("foo","bar","-1")
-config.setSectionKey("foo","foo","abc")
-config.writeConfig("test.ini")
-
-# test.ini now contains
-# [foo]
-# bar=-1
-# foo=abc
-
-var config2 = loadConfig("test.ini")
-let bar = config2.getSectionValue("foo","bar")
-let foo = config2.getSectionValue("foo","foo")
-assert(bar == "-1")
-assert(foo == "abc")
-echo "OK"
+import parsecfg, streams
+
+when not defined(js):
+  # bug #6046
+  var config = newConfig()
+  config.setSectionKey("foo", "bar", "-1")
+  config.setSectionKey("foo", "foo", "abc")
+
+  config.writeConfig("test.ini")
```

Comment:
pre-existing to ok to defer:

instead of polluting current dir (and complicating .gitignore), I prefer this:
```nim
from stdtest/specialpaths import buildDir
const file = buildDir/"test.ini"
config.writeConfig(file)
...
var config2 = loadConfig(file)
```